client/my-sites/plan-features-comparison/style.scss

```diff
@@ -182,6 +184,9 @@ $plan-features-sidebar-width: 272px;
 
 		.plan-features-comparison__header-annual-discount {
 			color: var( --studio-green-60 );
+			&-is-loading {
+				@include onboarding-placeholder();
```

Comment:
This works; tho there's also some pulsating loading component in Calypso components.

Reply:
Yep, defined here: https://github.com/Automattic/wp-calypso/blob/5e0a07e0ddaff970249d36123ff90e940bbacee0/client/assets/stylesheets/shared/mixins/_placeholder.scss#L6

It's probably already in scope so you won't need to import from other packages.